UltiSnips/javascript_jasmine.snippets

```diff
@@ -16,169 +12,58 @@ it('${1:description}', function() {
 });
 endsnippet
 
-snippet bef "before each (js)" b
-beforeEach(function() {
-	$0
-});
-endsnippet
-
-snippet aft "after each (js)" b
-afterEach(function() {
-	$0
-});
-endsnippet
-
-snippet any "any (js)" b
-jasmine.any($1)
-endsnippet
-
-snippet ru "runs (js)" b
-runs(function() {
-	$0
-});
-endsnippet
-
-snippet wa "waits (js)" b
-waits($1);
-endsnippet
-
-snippet ex "expect (js)" b
```

Comment:
they are used very often in jasmine tests - I approve that the triggers are short. `ee` makes more sense for expect to equal than expe, so I do not agree your suggested change is ergonomical.